client/my-sites/checkout/composite-checkout/components/item-variation-picker/variant-dropdown-price.tsx

```diff
@@ -244,10 +245,12 @@ export const ItemVariantDropDownPrice: FunctionComponent< {
 				) }
 			</Label>
 			<PriceTextContainer>
-				{ discountPercentage > 0 && ! isJetpack && ! isMobile && ! isIntroductoryOffer && (
-					<DiscountPercentage percent={ discountPercentage } />
-				) }
-				{ discountPercentage > 0 && ! isJetpack && ! isIntroductoryOffer && (
```

Comment:
Yep, it was a design request because we were avoiding displaying the discount and old price in other places. Context: p1671636858945399-slack-C03RCHNVC0H

Reply:
A related link: https://github.com/Automattic/wp-calypso/pull/71937